doc_src/en/messages.xml

```diff
@@ -0,0 +1,3 @@
+<entries>
+  <entry id="appendices">Appendices</entry>
```

Comment:
Right, this file contains labels to be included in the help, that are not explicit in doc sources neither in whc labels.

Note that some labels are included by whc which has its own localization. Whc localization does not include as much languages than OmageT help, this could cause an issue in the future.

Reply:
OK, thanks.